proto.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I'm assuming this is just a scratch file and will be deleted eventually? Otherwise, the intention is not clear what it's meant for.